tests/PHPStan/Rules/Functions/CallToFunctionParametersRuleTest.php

```diff
@@ -767,7 +767,7 @@ public function testProcOpen(): void
 
 		$this->analyse([__DIR__ . '/data/proc_open.php'], [
 			[
-				'Parameter #1 $command of function proc_open expects array<int, string>|string, array<string, string> given.',
+				'Parameter #1 $command of function proc_open expects array<int<0, max>, string>|string, array(\'something\' => \'bogus\', \'in\' => \'here\') given.',
```

Comment:
not sure why the array part at the end of the message now can see the concrete keys, instead of only `string` - not sure this is a intended side-effect